plugin.cpp

```diff
@@ -43,6 +43,15 @@ static const char *default_config = QUOTE({
 			"default" : "Random",
 			"order": "1",
 			"displayName": "Asset Name"
+			},
+		"numReadingsPerPoll" : {
+			"description" : "Number of values to be returned per poll call",
+			"type" : "integer",
+			"default" : "1",
+			"minimum" : "1",
+			"maximum" : "100000",
+			"order": "3",
+			"displayName": "Values Per Call"
```

Comment:
Please update description and displayName also because these will be displayed in GUI.